recipes/newrelic/apm/php/debian.yml

```diff
@@ -520,7 +534,7 @@ install:
               #
               target_ini_dir="/etc/php5/conf.d/"
               if [ -x /usr/sbin/phpquery ]; then
-                target_ini_dir="${ini}/../../mods-available/"
+                target_ini_dir="${php_ini_dir}/../mods-available/"
```

Comment:
explained in the second point [here](https://github.com/newrelic/open-install-library/pull/1131/files#r1837138349)